pkg/grpc/api.go

```diff
@@ -161,7 +162,12 @@ func (a *api) PublishEvent(ctx context.Context, in *dapr_pb.PublishEventEnvelope
 		body = in.Data.Value
 	}
 
-	envelope := pubsub.NewCloudEventsEnvelope(uuid.New().String(), a.id, pubsub.DefaultCloudEventType, body)
+	corID, ok := ctx.Value(tracing.CorrelationID).(string)
```

Comment:
Lines 165 - 169 should be moved to `diagnostics`

Reply:
Creating a method to wrap a couple of lines code with only one consumer seems to be unnecessary.

Reply:
Since this is using the `diagnostics` pkg and not `opencensus`, I *guess* it's ok..